RxSwift/Traits/Infallible/Infallible+CombineLatest+arity.swift

```diff
@@ -29,6 +29,24 @@ extension Infallible {
     }
 }
 
+extension InfallibleType where Element == Any {
```

Comment:
Is this constraint actually needed? Can't you just extend all InfallibleTypes ?

Reply:
@freak4pc Probably not, I just copied this from the `Observable` version, which has the constraint. Removing.